defender/forta_ctoken/serverless.yml

```
@@ -0,0 +1,79 @@
+service: generated-service
+configValidationMode: error
+frameworkVersion: "3"
+
+provider:
+  name: defender
+  stage: dev
+  stackName: forta_ctoken
+  ssot: false
+
+custom:
+  config: ${file(../secrets.yml)}
+  stackSecrets: ${self:custom.config.secrets.${self:provider.stackName}}
+
+  # Configure this section and rename secret-example.yml to secrets.yml
+  name: Forta cToken
+
+  # Forta Sentinel Settings
+  alert-ids:
+    - AE-COMP-CTOKEN-EVENT
+  agent-ids:
+    - "0x125c36816fbad9974a452947bf6a98d975988ddf4342c159a986383b64765e22"
```

I don't expect the Forta Sentinel to be re-enabled, given that the functionality is provided by a Contract Sentinel and Autotask, but the Bot ID is incorrect.
```suggestion
    - "0xab39733ddf86340b8e7940ebb933bb48506a341485c7f8c77da17bc1400fe028"
```